libr/include/r_io.h

```diff
@@ -307,6 +296,7 @@ typedef struct r_io_bind_t {
 	RIOClose close;
 	RIOReadAt read_at;
 	RIOWriteAt write_at;
```

Comment:
should not be used at all imho. bin plugins should just parse stuff, not modify the underlying data contents.

Reply:
the entire rbin architecture leaves a lot to be desired

Reply:
Agree, but we can always make it worse. I mean clean it up